test/isSet.js

```diff
@@ -0,0 +1,24 @@
+import * as RA from '../src';
+import eq from './shared/eq';
+import Symbol from './shared/Symbol';
+import args from './shared/arguments';
+
+describe('isSet', function() {
```

Comment:
Can you pls aligned your test with the new conventions we have for writing tests demonstrated here? https://github.com/char0n/ramda-adjunct/blob/9cd86e7d06de667c14f4fdc8661d0fd4a4c99dc3/test/isNotBoolean.js#L5

We also missing the test for currying completely.

Reply:
Moved it to the new standard. I am confused about testing for currying. I tried to check other examples where it was used, for example isNotBoolean, however, their tests don't have anything that jumps out as different from what I wrote. Could you elaborate if I got it incorrect? Thank you

Reply:
It's possible that I overlook this test `should support placeholder to specify "gaps"`. Now i can see it, so marking this as resolved.